app/(saas)/quests/_features/quest-list/quest-list.tsx

```
@@ -6,11 +6,25 @@ import { NEXT_ROUTER } from "@/shared/constants/router";
 import { QuestCard } from "../../_components/quest-card/quest-card";
 
+const statusPriority = {
+  started: 0,
+  "application-open": 1,
+  finished: 2,
+};
+
 export function QuestList() {
   const router = useRouter();
-  const quests = QuestListData;
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)